app/services/importer_service.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class ImporterService
+  def initialize(data_set_id, fields: {})
+    @data_set_id = data_set_id
+    @fields = fields
+    @per_page = 500
```

Comment:
Why set this here if it can't be overridden? Either take it as an argument or set it as a constant

Reply:
because it needs to be read several time -- and i needed to reduce the complexity of the `import!` function